samples/net/lwm2m_client/overlay-dtls-cert.conf

```diff
@@ -0,0 +1,42 @@
+CONFIG_LWM2M_DTLS_SUPPORT=y
```

Comment:
Shouldn't this overlay also enforce bootstrap? We're not feeding any certificates from the application.

Reply:
Not necessary.. I was testing this manually by feeding the certificates from command line.

```
lwm2m write 0/0/3 -s "-----BEGIN CERTIFICATE-----\x0aMIIBazCCAR...
lwm2m write 0/0/5 -s "-----BEGIN EC PRIVATE KEY-----\x0aMHcCAQE...
lwm2m write 0/0/4 -s "-----BEGIN CERTIFICATE-----\x0aMIICBzCCAa...
lwm2m write 0/0/2 -u8 2
lwm2m start secure_client
```

I prefer to write separate documentation as there are now 3 different security modes supported. But I would like to do that on another commit.

Reply:
Yes, an update in the documentation would be much appreciated. I'm ok with doing this separately.